### PR TITLE
Update PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by lpil**
### Motivation

Let's make PRs better and help avoid premature merges.

### Test plan

N/A

### Pre-flight checklist

- [ ] Team review

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,10 +1,26 @@
...
+- [ ] Manual testing
+  - [ ] Latest Chrome
+  - [ ] Latest Firefox
+  - [ ] IE11
+  - [ ] Edge
+  - [ ] Latest Safari
+  - [ ] iPhone 6
+  - [ ] Google Nexus 5
+  - [ ] Samsung S6
+  - [ ] iPad (5th gen)
```
Comment (asavin) [inline .github/PULL_REQUEST_TEMPLATE.md]: Should we instead of 5th gen specify iOS version here?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,10 +1,26 @@
...
+  - [ ] Design review
+- [ ] Manual testing
+  - [ ] Latest Chrome
+  - [ ] Latest Firefox
+  - [ ] IE11
+  - [ ] Edge
+  - [ ] Latest Safari
+  - [ ] iPhone 6
+  - [ ] Google Nexus 5
+  - [ ] Samsung S6
```
Comment (asavin) [inline .github/PULL_REQUEST_TEMPLATE.md]: Android version here and for the Nexus?

**Comment by asavin** [comment]
We're not supporting Android native browser anywhere, right? Chrome only?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (lpil) [inline .github/PULL_REQUEST_TEMPLATE.md]: I'm only copying up what is currently on the board. The new list will come later.

**Pull request merged**